CREATE TABLE produkt

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE produkt 
+(id_produktu bigserial PRIMARY KEY,
+ nazwa VARCHAR(50) UNIQUE NOT NULL,
+kategoria VARCHAR(50) NOT NULL,
+ilosc SMALLINT NOT NULL,
+cena NUMERIC(7,2) NOT NULL,
+rozmiar VARCHAR(20) NOT NULL,
+polozenie VARCHAR(20) NOT NULL,
+data_modyfikacji timestamp NOT NULL)
+
+
+CREATE TABLE zespol 
+(id_zespolu serial PRIMARY KEY,
+ kierownik VARCHAR(50) NOT NULL,
+zmiana VARCHAR(50) NOT NULL)
+
+
+CREATE TABLE klient 
+(id_klienta bigserial PRIMARY KEY,
+ imie VARCHAR(50) NOT NULL,
+nazwisko VARCHAR(50) NOT NULL,
+email VARCHAR(50) UNIQUE NOT NULL CHECK (email ~* '^[A-Za-z0-9._%-]+@[A-Za-z0-9.-]+[.][A-Za-z]+$'),
+telefon VARCHAR(20) ,
+adres VARCHAR(50) NOT NULL,
+met_platnosci VARCHAR(50) NOT NULL,
+data_utworzenia timestamp NOT NULL)
+
+
+CREATE TABLE pracownik 
+(id_pracownika bigserial PRIMARY KEY,
+ id_zespolu INTEGER REFERENCES zespol (id_zespolu) NOT NULL,
+pozycja VARCHAR(50) NOT NULL,
+imie VARCHAR(50) NOT NULL,
+nazwisko VARCHAR(50) NOT NULL,
+data_utworzenia timestamp NOT NULL)
+
+
+CREATE TABLE koszyk 
+(id_koszyka bigserial PRIMARY KEY,
+ id_klienta INTEGER REFERENCES klient(id_klienta) NOT NULL,
+id_produktu0 INTEGER REFERENCES produkt(id_produktu) NOT NULL,
+ilosc0 SMALLINT NOT NULL,
+id_produktu1 INTEGER REFERENCES produkt(id_produktu),

```

Comment:
Having hard-coded products in a table like this may turn out to be very problematic, because not only we limit the amount of Produkty in Koszyk and changing such limit will require a database schema change, but it'll also be quite problematic when dealing with logic of deleting and adding products.

For example, if Klient has 9 Produkty in his Koszyk and he decided to delete 3rd and 5th Produkty and then decides to add some different product back again, then we'd have to iterate over every of those columns to find an empty column and add it here. This would be inconvenient to maintain.